wrappers/services.go

```diff
@@ -332,6 +347,8 @@ WantedBy={{.ServicesTarget}}
 		PrerequisiteTarget: systemd.PrerequisiteTarget,
 		MountUnit:          filepath.Base(systemd.MountUnitPath(appInfo.Snap.MountDir())),
 		Remain:             remain,
+		Before:             genServiceNames(appInfo.Snap, appInfo.Before),
+		After:              genServiceNames(appInfo.Snap, appInfo.After),
```

Comment:
Is appInfo.Before / appInfo.After already validated at this point to contain actual service names only? I think we need to make sure we clearly manifest any mismatches and not drop them silently.

Reply:
Yes, this is called after the validation part, so whatever is in Before/After lists must be a valid app in the snap.

If you feel like doing more reviews, the validation pieces are in #4373